prep-javascript-variables/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+
+<body>
+  <script>
+    var fullName = "John Hwang";
+    console.log("value of fullName: " + fullName);
+    var numberOfPets = 0;
+    console.log("value of numberOfPets: " + numberOfPets);
+    var likesVegetables = true;
+    console.log("value of likesVegetables: " + likesVegetables);
+    var nothing = null;
+    console.log("value of nothing: " + nothing);
+    var never;
+    console.log("value of never: " + never);
```

Comment:
Here you should use a `,` comma instead of a `+` plus, because certain values, like objects and arrays, will get messed up if you combine them with your strings.